config.go

```diff
@@ -112,6 +113,11 @@ func (b *Builder) fixupConfig() {
 	if b.Format == define.Dockerv2ImageManifest && b.Hostname() == "" {
 		b.SetHostname(stringid.TruncateID(stringid.GenerateRandomID()))
 	}
+	if b.Logger == nil {
+		b.Logger = logrus.New()
+		b.Logger.SetOutput(os.Stderr)
+		b.Logger.SetLevel(logrus.GetLevel())
+	}
```

Comment:
This snippet belongs somewhere other than in a method called `fixupConfig()`.  It's also redundant for callers who get here from `newBuilder()`.

Reply:
Where would you want me to put it.  fixupConfig is called by both newbuilder and openbuilder. I could make a new function? And call it from both?

Reply:
OpenBuilder is called from buildah run ...

Reply:
It's either going to be do-it-everywhere, or a helper function.  Builder objects are also constructed by the variations of OpenBuilder, and by ImportBuilder/ImportBuilderFromImage.  Basically, anywhere you see a new Builder object being created, we're going to need to set this field.